src/runtime_src/core/pcie/tools/xbmgmt/cmd_hotplug.cpp

```diff
@@ -114,6 +134,46 @@ int hotplugHandler(int argc, char *argv[])
     return ret;
 }
 
+static int shutdownDevice(const std::shared_ptr<pcidev::pci_device> dev)
+{
+    std::string errmsg;
+    int shutdownStatus = -EINVAL;
+    int wait  = 0;
+
+    /* "echo 1 > /sys/bus/pci/<EndPoint>/shutdown" to trigger shutdown of the device */
+    std::string path = dev->get_sysfs_path("", "shutdown");
+    if (!boost::filesystem::exists(path)) 
+    {
+        return -EEXIST;
```

Comment:
shouldn't you return ENOENT?

Reply:
Thanks for the comments.  I will make the changes and publish them again.